### PR TITLE
us01 is 100% done

### PR DESCRIPTION
**Pull request by dilovar1992**

### CODE REVIEW

File: src/test/java/com/trycloud/pages/DashboardPage.java [hunk 1]
```diff
@@ -7,35 +7,49 @@
...
+
+    //other elements
+    @FindBy(xpath = "(//button)[3]")
+    public WebElement closeButton;
+    @FindBy(xpath = "//a[.='Customize']")
+    public WebElement customizeButton;
+
+
     //return webelement of modules
     public WebElement modules(String module) {
```
Comment (ersuamine) [inline src/test/java/com/trycloud/pages/DashboardPage.java:44]: This is basically the same method present in the base page

File: src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java [hunk 2]
```diff
@@ -28,7 +32,7 @@ public void user_should_be_able_to_see_following_modules(List<String> expectedMo
             //then assert isDisplayed or not
             assertTrue(dash.modules(expectedModule).isDisplayed());
         }
-        //i could not extract text, why?
+        //I could not extract text, why?
```
Comment (ersuamine) [inline src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java:35]: module element is the "a" tag and it does not have the text name of the module, if you want to extract text, you should target the child span tag under the "a" tag

File: src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java [hunk 3]
```diff
@@ -50,8 +54,28 @@ public void userShouldBeAbleToSee(String expectedUserName)  {
     public void user_clicks_on_customize_button() {
        dash.customizeButton.click();
     }
-    @Then("user can select any {string}")
-    public void user_can_select_any(String widget) {
-        dash.TalkMentions.click();
+
+    @Then("user can select any widget like {string}")
+    public void user_can_select_any_widget_like(String widget) {
+        dash.chooseWidget(widget);
```
Comment (ersuamine) [inline src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java:60]: dash?

**Review by ersuamine — APPROVED** [review]

File: src/test/java/com/trycloud/pages/DashboardPage.java (hunk 1 above)
Comment (dilovar1992) [reply src/test/java/com/trycloud/pages/DashboardPage.java:44]: Nope the base one clicks mine just used to create the weblelemnt

File: src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java (hunk 2 above)
Comment (dilovar1992) [reply src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java:35]: It doesn't work, i tried many ways from many parents , but to no veil

File: src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java (hunk 3 above)
Comment (dilovar1992) [reply src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java:60]: Object of dashbord page

**Pull request merged**